conf/azurebatch.config

```diff
@@ -0,0 +1,27 @@
+//Nextflow config file for running on Azure batch
+params {
+  config_profile_description = 'AWSBATCH Cloud Profile'
+  config_profile_contact = 'Venkat Malladi (@vsmalladi)'
+  config_profile_url = 'https://azure.microsoft.com/services/batch/'
+
+  az_location = 'westus2'
+  batch_name = false
+  batch_key = false
+  storage_name = 'platinumgenomes'
+  sas_token = 'sv=2019-02-02&se=2050-01-01T08%3A00%3A00Z&si=prod&sr=c&sig=FFfZ0QaDcnEPQmWsshtpoYOjbzd4jtwIWeK%2Fc4i9MqM%3D'
+  work_dir = false
+}
+
+docker {
+  enabled = true
+}
+
+
+azure.batch.location = params.az_location
+azure.batch.accountName = params.batch_name
+azure.batch.accountKey = params.batch_key
+azure.batch.autoPoolMode = true
+azure.storage.accountName = params.storage_name
+azure.storage.sasToken = params.sas_token
+process.executor = 'azurebatch'
+workDir = params.work_dir
```

Comment:
Why not assume that people will run with `-w` or `-work-dir`? Having `--work_dir` as well strikes me as being a little confusing.. I think it might be clearer to leave this out and just document it as necessary in the readme.

Reply:
Good idea. Will update